File: code_forces/69A/A.java
import java.util.*;
import java.io.*;

public class A {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int ans = 0;
        int[][] arr = new int[n][3];

        for(int i=0; i<n; i++) {
            for(int j=0; j<3; j++) {
                int temp = sc.nextInt();
                arr[i][j] = temp; 
            }
        }

        for(int j=0; j<3; j++) {
            for(int i=0; i<n; i++) {
                ans+=arr[i][j];
            }
            if(ans != 0) {
                System.out.println("NO");
                System.exit(0);
            }
            ans = 0;
        }
        System.out.println("YES");
    }
}

File: code_forces/69A/a.py
import sys

def check_sum(arr, n):
    ans=[]
    for j in range(3):
        for i in range(n):
            ans.append(arr[i][j])
        if sum(ans) != 0:
            return 'NO'
        ans.clear()

    return 'YES'


def main():
    n = int(sys.stdin.readline())
    vector = []
    for i in range(n):
        temp = list(map(int, sys.stdin.readline().split()))
        vector.append(temp)

    sys.stdout.write(f'{check_sum(vector, n)}')


if __name__ == '__main__':
    main()